src/main/java/edu/wpi/first/shuffleboard/sources/SourceType.java

```diff
@@ -9,7 +9,7 @@
 public enum SourceType {
 
   NONE(false, "", null),
-  STATIC(false, "example://", s -> DummySource.forTypes(DataType.forName(s)).get()),
+  STATIC(false, "example://", s -> DummySource.forTypes(DataType.forName(s)).orElse(null)),
```

Comment:
because this makes it throw the error in forURI rather then here. Looking at it now though, it could probably return a DataSource.none()